### DISCUSSION TITLE
Aquacomputer OCTO

### DISCUSSION THREAD

**Discussion by mitro0609** [Rem0o/FanControl.Releases#922] [Q&A]
I have really enjoyed using FanControl on my computers (and others' computers) , But I recently went to a custom loop on one of them and needed a controller for the additional fans so I decided to get and OCTO to control them because it seemed to be the most recommended. I assumed that Fan Control would work with it and I have not been able to get it to work.

FanControl won't control the OCTO. I've tried multiple times. It DOES see the fans (speeds), but doesn't show the corresponding controls and won't control them. First, I tried before even installing Aquacomputer's software, aquasuite (which is far more complicated to even get close to what Fan Control does). **I always made sure that I selected "Controller (various third parties)" in sensor settings**. Then I tried after installing Aquacomputer's software with the same result. Then I made sure the aquasuite service was shut down and deleted my FanControl folder to start fresh and ran though the assisted setup with same result: no control of the attached fans, but can see speeds.

Is there anyone who has FanControl working with an OCTO, and if so what could I be doing wrong? There's only one mention of the OCTO in the discussions and it just says that it "should" work.

Thanks in advance.

EDIT: I have 7 fans and one Corsair water pump connected to the fan headers and 1 temp probe (coolant temp from corsair pump) also connected.

**Comment by Rem0o** [comment]
https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/commit/b0de6e3ad762c18083b9198bd1982537e7c0f16c#diff-7519a33c367a41fdb1c141a6e7deb68d6c1a76b09ffe472e85dbe0e40681af60

From what I can see, the octo support is read-only, no fan control.
